pontoon/base/signals.py

```diff
@@ -17,3 +22,78 @@ def project_locale_added(sender, **kwargs):
         project = project_locale.project
         project.has_changed = True
         project.save(update_fields=['has_changed'])
+
+
+@receiver(pre_save, sender=Locale)
+def create_locale_translators_group(sender, **kwargs):
+    """
+    Creates respective translators group in django's permission system.
+    Managers should be able to translate and manage locale.
+    """
+    instance = kwargs['instance']
+
+    if kwargs['raw'] or instance.translators_group:
+        return
+    try:
+        locale_ct = ContentType.objects.get(app_label='base', model='locale')
+        locale_group, _ = Group.objects.get_or_create(name='{} translators'.format(instance.code.upper()))
+        can_translate = Permission.objects.get(content_type=locale_ct, codename='can_translate_locale')
+        locale_group.permissions.add(can_translate)
+        instance.translators_group = locale_group
+    except:
```

Comment:
Yeah, i know it's called "pokemon-style" catching of exceptions and in 99% of cases i wouldn't use it. I had version with ObjectDoesNotExist, but i had second thoughts. What if code inside changes in a way that cause different exception and site still will be broken? Everything will be logged into raygun so it won't be silenced in any way. Maybe i'm overthinking it :thought_balloon: